qcore/binary_version.py

```diff
@@ -0,0 +1,38 @@
+"""
+A class to retrieve versioned files.
+Defaults are available from the config files
+"""
+
+from config import qconfig
```

Comment:
from qcore.config import qconfig otherwise might break?

Reply:
Should include this in the consistent coding guide?